robottelo/config/validators.py

```diff
@@ -0,0 +1,251 @@
+from dynaconf import Validator
+
+from robottelo.constants import AZURERM_VALID_REGIONS
+from robottelo.constants import VALID_GCE_ZONES
+
+
+validators = dict(
+    server=[
+        Validator("server.hostname", must_exist=True),
+        Validator("SERVER.ssh_key", must_exist=True)
```

Comment:
s/SERVER/server/ ? 

and below as well ?

Reply:
Guess that was added before Bruno introduced case-insensitive keys into dynaconf. I believe now dynaconf understands both, so it doesn't matter for it.
Changed, thanks.